gtsam/geometry/SOn.h

```diff
@@ -30,6 +30,9 @@
 #include <vector>
 #include <random>
 
+ // For save/load
+#include <boost/serialization/split_member.hpp>
```

Comment:
Why? Does vanilla serialization not work?

Reply:
My bad, I did this to fix a boost bug revealed by the new test matrix, but forgot which bug it is...

Reply:
Not understanding this comment. The change was made by @mikesheffler, right? Also, if it is *not* needed, we should not do it - as it adds code and mystery.

Reply:
Both are true! It's Fan's code; I just replicated it exactly as it was on the `feature/new_wrapper` branch.